app/Http/Controllers/JobController.php

```diff
@@ -2,9 +2,97 @@
 
 namespace App\Http\Controllers;
 
+use App\Models\Job;
 use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
 
 class JobController extends Controller
 {
-    //
+    /**
+     * Show job creation form.
+     *
+     * @return \Illuminate\Contracts\Support\Renderable
+     */
+    public function create()
+    {
+        return view('jobs.create');
+    }
+
+    /**
+     * Store job.
+     *
+     * @return \Illuminate\Contracts\Support\Renderable
+     */
+    public function store(Request $request)
+    {
+
+        $formFields = $request->validate([
+            'keywords' => 'required',
+            'title' => 'required',
+            'description' => 'required',
```

Comment:
maybe worth to add the max length for some of these fields, in case they are simple varchar it might be like 
```
'description' => 'required|string|max:255',
```